api/seccompprofile/v1beta1/seccompprofile_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Kubernetes Authors.
+Copyright 2020-2022 The Kubernetes Authors.
```

Comment:
We can leave this as is :) 
```suggestion
Copyright 2020 The Kubernetes Authors.
```